drivers/audio/Kconfig

```diff
@@ -238,7 +238,7 @@ config WM8904_NUM_BUFFERS
 
 config WM8904_WORKER_STACKSIZE
 	int "WM8904 worker thread stack size"
-	default 768
```

Comment:
This is a horrendous increase and very WRONG!
It is inappropriate to increase the size of this tuned stack size.  Let's not do that.